<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveDialog</class>
 <widget class="QWidget" name="SaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="saveBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>323</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Closing device connection you will lose all data received
do you want to save them in a file?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
